core/src/main/java/hudson/security/csrf/CrumbIssuer.java

```diff
@@ -43,6 +43,7 @@
 public abstract class CrumbIssuer implements Describable<CrumbIssuer>, ExtensionPoint {
 
     private static final String CRUMB_ATTRIBUTE = CrumbIssuer.class.getName() + "_crumb";
+    public static final String DEFAULT_CRUMB_NAME = "Jenkins-Crumb";
```

Comment:
Why not move it to `DefaultCrumbIssuer`?

In either case, it need not be `public`.